lmfdb/artin_representations/math_classes.py

```diff
@@ -179,8 +179,9 @@ def power_prime(p, exponent):
                 return " " + str(p) + " "
             else:
                 return " " + str(p) + "^{" + str(exponent) + "}"
-        tmp = r" \cdot ".join(power_prime(p, val) for (p, val) in self.factored_conductor())
-        return tmp
+
+        return r" \cdot ".join(power_prime(p, val)
+                               for p, val in self.factored_conductor())
```

Comment:
I find that this line break makes the code less readable (the same is true for many instances below).